ctms/schemas/bulk.py

```diff
@@ -0,0 +1,54 @@
+import base64
+from datetime import datetime, timezone
+from typing import Literal, Optional, Union
+
+from pydantic import validator
+
+from .base import ComparableBase
+
+BLANK_VALS = [None, "", "null"]  # TODO: Should "null" be allowed?
+
+
+class BulkRequestSchema(ComparableBase):
+    """A Bulk Read Request."""
+
+    start_time: datetime
+
+    end_time: Optional[Union[datetime, Literal[""]]] = None
+
+    @validator("end_time")
+    def end_time_must_not_be_blank(cls, value):  # pylint: disable=E0213
```

Comment:
E0213 is the warning "E0213: Method should have "self" as first argument (no-self-argument)". I have a preference for the string version, ``# pylint: disable=no-self-argument``, but it shorter and just a search away.

I checked if we're doing something wrong, but this seems to be a [bad interaction between pylint and pydantic being compiled with cython](https://github.com/samuelcolvin/pydantic/issues/568).

Reply:
Changed to the string version: resolving